src/measure_theory/l1_space.lean

```diff
@@ -16,24 +58,72 @@ set_option class.instance_max_depth 100
 namespace measure_theory
 open set lattice filter topological_space ennreal emetric
 
-universes u v
-variables {α : Type u} {β : Type v} [measure_space α]
-variables {γ : Type*} [normed_group γ]
+universes u v w
+variables {α : Type u} [measure_space α]
+variables {β : Type v} [normed_group β]
+variables {γ : Type w} [normed_group γ]
 
-infixr ` →ₘ `:25 := ae_eq_fun
+def integrable (f : α → β) : Prop := (∫⁻ a, nnnorm (f a)) < ⊤
 
-def integrable (f : α → γ) : Prop := (∫⁻ a, nnnorm (f a)) < ⊤
+lemma integrable_of_ae_eq {f g : α → β} (hf : integrable f) (h : ∀ₘ a, f a = g a) : integrable g :=
+begin
+  simp only [integrable] at *,
+  have : (∫⁻ (a : α), ↑(nnnorm (f a))) = (∫⁻ (a : α), ↑(nnnorm (g a))),
+    apply lintegral_congr_ae,
+    filter_upwards [h],
+    assume a,
+    simp only [mem_set_of_eq],
+    assume h,
+    rw h,
```

Comment:
If this is proving the `have : ...` statement, can you enclose this in braces? (It helps to know where a proof starts and is supposed to end, especially when the proof breaks and needs fixing).